src/components/Pagination/Pagination.st.css

```diff
@@ -0,0 +1,264 @@
+
+:import {
+  -st-from: "wix-ui-core/index.st.css";
+  -st-named: Pagination
+}
+
+:import {
+    -st-from: "../../tpaStyles/styleableExt";
+    -st-named: wixApply;
+}
+
+:vars {
+  /* PAGES MODE */
+  selectedPageBG: transparent;
+  selectedPageBorderColor: transparent;
+  selectedPageBorderWidth: 0;
+  selectedPageTextColor: #ed24d9;
+  totalPagesTextColor: #000;
+  pageNumbersTextColor: #000;
+
+
+  /* NAV BUTTONS GENERAL */
+  navButtonBG: transparent;
+  navButtonBorderWidth: 3;
+  navButtonBorderColor: transparent;
+  navButtonDisabledBorderWidth: 3;
+  navButtonDisabledBorderColor: transparent;
+  navButtonDisabledBG: transparent;;
+
+
+  /* ICON NAV BUTTONS */
+  arrowsColor: #000;
+  arrowsDisabledColor: #b1b1b1;
+
+  /* TEXT NAV BUTTONS */
+  navButtonTextSize: 15px;
+  navButtonTextColor: #000;
+  navButtonDisabledTextColor: #b1b1b1;
+
+  /* @type FONT */
+  fnt: font_8;
+  bRadius: 0;
+  shadow: none;
+}
+
+:vars {
+  TextFont: --overridable;
+  TextFontColor: --overridable;
+  SelectedTextFontColor: --overridable;
+}
+
+.pageButtonRules {
+  min-width: 1em;
+  min-height: 1em;
+  line-height: 1em;
+  justify-content: center;
+  align-items: center;
+  padding: 0.5em 0.5em;
+  margin: 0.25em;
+}
+
+.root {
+  -st-extends: Pagination;
+  -st-states: rtl, paginationMode(enum(pages, input)), navButtonsType(enum(text, arrows)), navButtonsAmount(number), alignment(enum(center, left, right));
+  align-items: stretch;
+  box-sizing: border-box;
+  display: flex;
+  font:  wixApply(fallback, wixApply(font, value(TextFont)), value(fnt));
+}
+
+.root:alignment(left) {
+  justify-content: flex-start;
+}
+
+.root:alignment(center) {
+  justify-content: center;
+}
+.root:alignment(right) {
+  justify-content: flex-end;
+}
+
+.root::pageButton {
+  -st-mixin: pageButtonRules;
+  position: relative;
+  color: wixApply(color, wixApply(fallback, value(TextFontColor), value(pageNumbersTextColor)));
+}
+
+.root::currentPage {
+  -st-mixin: pageButtonRules;
+  color: wixApply(color, wixApply(fallback, value(SelectedTextFontColor), value(selectedPageTextColor)));
+  background: value(selectedPageBG);
+  border-radius: value(bRadius);
+  border: value(selectedPageBorderWidth) solid value(selectedPageBorderColor);
+  box-shadow: value(shadow);
+  padding: calc(0.5em - value(selectedPageBorderWidth)) calc(0.5em - value(selectedPageBorderWidth));
+}
+
+.root:not(:disabled)::pageButton:hover::before {
+  background: value(selectedPageBG);
+  opacity: 0.2;
+  content: "";
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  border-radius: value(bRadius);
+}
+
+.root:not(:disabled)::pageButton:hover {
+  opacity: 0.7;
+}
+
+.root:disabled::pageButton {
+  cursor: default;
+}
+
+.root::navButton {
+  min-width: 1em;
+  min-height: 1em;
+  padding: calc(0.75em - value(navButtonBorderWidth));
```

Comment:
I think either have `navButtonBorderWidth: 3px`, or `padding: calc(0.75em - value(navButtonBorderWidth)px);`
Or am I missing something?

Reply:
Same for all places we need a `px`-ed value